proto/dex/query.proto

```diff
@@ -112,23 +121,25 @@ message QueryAllShortBookResponse {
 }
 
 message QueryGetSettlementsRequest {
-	string contractAddr = 1;
```

Comment:
we can, but even if it's in snakecase we still need to add `[(gogoproto.jsontag) = "twaps"]` for it to be encoded into json properly, since most of our structs use gogoproto

Reply:
hm, some of the oracle protos don't need that and are still generated properly 🤔

Reply:
oh wait, some of those do have `omitempty`